index.html

```diff
@@ -217,7 +217,7 @@
       <section class="main-section" id="reference">
         <header>Reference</header>
```

Comment:
For your final touches:
- Consider replacing <header> tags inside sections with <h1> through to <h6>. You can choose your range here 😄 .

> Read up more on the importance of these (here)[https://www.w3.org/WAI/tutorials/page-structure/headings/] or on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)